2023-02-27 11:34:27,983 INFO    MainThread:35176 [wandb_setup.py:_flush():68] Configure stats pid to 35176
2023-02-27 11:34:27,984 INFO    MainThread:35176 [wandb_setup.py:_flush():68] Loading settings from C:\Users\juddb\.config\wandb\settings
2023-02-27 11:34:27,984 INFO    MainThread:35176 [wandb_setup.py:_flush():68] Loading settings from C:\UCF\Projects\Multus Medium\movie rec prot\RecSystem\wandb\settings
2023-02-27 11:34:27,984 INFO    MainThread:35176 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-02-27 11:34:27,984 INFO    MainThread:35176 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'C:\\UCF\\Projects\\Multus Medium\\movie rec prot\\RecSystem\\main.py'}
2023-02-27 11:34:27,984 INFO    MainThread:35176 [wandb_init.py:_log_setup():478] Logging user logs to C:\UCF\Projects\Multus Medium\movie rec prot\RecSystem\wandb\run-20230227_113427-8c6390np\logs\debug.log
2023-02-27 11:34:27,985 INFO    MainThread:35176 [wandb_init.py:_log_setup():479] Logging internal logs to C:\UCF\Projects\Multus Medium\movie rec prot\RecSystem\wandb\run-20230227_113427-8c6390np\logs\debug-internal.log
2023-02-27 11:34:27,985 INFO    MainThread:35176 [wandb_init.py:init():518] calling init triggers
2023-02-27 11:34:27,985 INFO    MainThread:35176 [wandb_init.py:init():521] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.01, 'epochs': 20, 'batch_size': 128, 'architecture': 'MF'}
2023-02-27 11:34:27,985 INFO    MainThread:35176 [wandb_init.py:init():571] starting backend
2023-02-27 11:34:27,985 INFO    MainThread:35176 [wandb_init.py:init():575] setting up manager
2023-02-27 11:34:28,008 INFO    MainThread:35176 [wandb_init.py:init():582] backend started and connected
2023-02-27 11:34:28,017 INFO    MainThread:35176 [wandb_init.py:init():670] updated telemetry
2023-02-27 11:34:28,020 INFO    MainThread:35176 [wandb_init.py:init():710] communicating run to backend with 60.0 second timeout
2023-02-27 11:34:31,093 WARNING MsgRouterThr:35176 [router.py:message_loop():77] message_loop has been closed
2023-02-27 11:34:33,097 ERROR   MainThread:35176 [wandb_init.py:init():1135] error
Traceback (most recent call last):
  File "C:\Users\juddb\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 1108, in init
    run = wi.init()
  File "C:\Users\juddb\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 713, in init
    result = run_init_handle.wait(
  File "C:\Users\juddb\AppData\Local\Programs\Python\Python39\lib\site-packages\wandb\sdk\lib\mailbox.py", line 271, in wait
    raise MailboxError("transport failed")
wandb.errors.MailboxError: transport failed
